main.nf

```diff
@@ -75,8 +75,8 @@ if (params.reads) {
         .splitCsv(header: true, sep: ',')
         .map{row ->
             def sample = row['Sample']
-            def read1 = path(row['R1'], checkIfExists: true)
```

Comment:
I am not entirely sure why, but I got errors when executing this as ``path()`` instead of ``file()``